pytest_repo_health/plugin.py

```diff
@@ -171,7 +171,7 @@ def pytest_collection_modifyitems(session, config, items):
                 # add doc string if function also has doc string
                 if item.function.__doc__ is not None:
                     checks_metadata[module_name][item.name]["doc_string"] = item.function.__doc__.strip()
-        with open(config.getoption("repo_health_metadata"), "w") as write_file:
+        with open(config.getoption("repo_health_metadata"), "w", encoding='utf-8') as write_file:
```

Comment:
Update the `pylintrc` file instead using the command `edx_lint write pylintrc`. For more context, see: https://github.com/edx/edx-lint#using-edx_lint.
The updated version has disabled this warning by default so you won't have to add this parameter.